samples/highcharts/accessibility/accessible-switch-theme/demo.css

```diff
@@ -0,0 +1,6 @@
+.container-column, .container-spline, .container-scatter, .container-map, .container-pie {
```

Comment:
Same as with pattern fill demo - gets a bit squeezed on mobile - collapse to single column?